### PR TITLE
HOTFIX: Adding in an admin tool for manually marking/unmarking validators

### PR DESCRIPTION
**Pull request by coachchucksol**

### CODE REVIEW

File: utils/steward-cli/src/commands/actions/surgery.rs [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+        transactions::configure_instruction,
+    },
+};
+
+pub async fn command_surgery(
+    args: Surgery,
+    client: &Arc<RpcClient>,
+    program_id: Pubkey,
+) -> Result<()> {
+    let validator_list_index: u64 = 0;
```
Comment (ebatsell) [inline utils/steward-cli/src/commands/actions/surgery.rs:27]: make this an arg? or hard code?

File: utils/steward-cli/src/main.rs [hunk 2]
```diff
@@ -42,15 +42,16 @@ async fn main() -> Result<()> {
...
-        }
+        // Commands::ViewNextIndexToRemove(args) => {
+        //     command_view_next_index_to_remove(args, &client, program_id).await
+        // }
 
         // --- Actions ---
         Commands::InitConfig(args) => command_init_config(args, &client, program_id).await,
         Commands::UpdateConfig(args) => command_update_config(args, &client, program_id).await,
-        Commands::InitState(args) => command_init_state(args, &client, program_id).await,
-        Commands::ResetState(args) => command_reset_state(args, &client, program_id).await,
```
Comment (ebatsell) [inline utils/steward-cli/src/main.rs:53]: why comment these out?

File: programs/steward/src/instructions/admin_mark_for_removal.rs [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    mark_for_removal: bool,
+    immediate: bool,
+) -> Result<()> {
+    let mut state = ctx.accounts.state_account.load_mut()?;
+
+    if immediate {
+        state
+            .state
+            .validators_for_immediate_removal
+            .set(validator_list_index, mark_for_removal)?;
```
Comment (ebatsell) [inline programs/steward/src/instructions/admin_mark_for_removal.rs:36]: should be `immediate`

File: utils/steward-cli/src/commands/actions/surgery.rs (hunk 1 above)
Comment (coachchucksol) [reply utils/steward-cli/src/commands/actions/surgery.rs:27]: arg

File: utils/steward-cli/src/main.rs (hunk 2 above)
Comment (coachchucksol) [reply utils/steward-cli/src/main.rs:53]: fixed

File: programs/steward/src/instructions/admin_mark_for_removal.rs (hunk 3 above)
Comment (coachchucksol) [reply programs/steward/src/instructions/admin_mark_for_removal.rs:36]: ?

**Review by ebatsell — APPROVED** [review]

**Pull request merged**